tests/bootstrap.php

```diff
@@ -3,6 +3,7 @@
 
 namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests;
 
+use DG\BypassFinals;
```

Comment:
This library is needed for bypassing final class. 

In v16 google marked all client services as Final. So when we mock them we got an error.

Please see https://github.com/googleads/google-ads-php/issues/1008